src/OpenTelemetry.Instrumentation.ElasticsearchClient/README.md

```diff
@@ -26,9 +29,18 @@ services.AddOpenTelemetryTracing(x =>
 });
 ```
 
+## Elastic.Clients.Elasticsearch
```

Comment:
Personally, I would move this section before "NEST/Elasticsearch.Net". Because "Elastic.Clients.Elasticsearch" is newer and this is what the users should use 😉

Reply:
I would keep it as is it is. Readme is strictly related to the package.
I would like to have just a note that it will not work with newer versions.

Reply:
I would like to resolve this comment, but I cannot 😉